SPECS/python-attrs/python-attrs.spec

```diff
@@ -39,7 +39,7 @@ Attributes without boilerplate.
 
 %check
 # explicitly  install packaging ver to match azl install to avoid pip conflict
-pip3 install tox packaging==23.2
+pip3 install tox 
```

Comment:
We have a `pip3 install packaging==23.2` in `python-gevent.spec`. Might be worth trying to replace it with a `BuildRequires: python3-packaging`.